### DISCUSSION TITLE
Does the user token get automatically validated every hour?

### DISCUSSION THREAD

**Discussion by jakjakob** [Teekeks/pyTwitchAPI#317] [Q&A] [answered]
As of https://dev.twitch.tv/docs/authentication/validate-tokens/, you need to validate the user token every hour. I am wondering if it is done automatically or if we need to implement it on our own

**Comment by Teekeks** [comment, marked as answer]
The lib does not validate once every hour but if it does auto renew the token once it realized that the token has expired.
